modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path

```diff
@@ -620,6 +620,11 @@
 		<td>//div[contains(@class,'card-body')]//button[contains(text(), '${key_optionDefaulValue}')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>VIEW_ICON_OBJECT_ADMIN</td>
+		<td>//div[(@class='table-list-title')]//*[(@class='lexicon-icon lexicon-icon-view')]</td>
+		<td></td>
+	</tr>
```

Comment:
I added a new path to click on `View Icon`.

The View Icon is available when changing the portal language and not adding a new translation to the object/field.

![Screenshot](https://github.com/liferay-bpm-qa/liferay-portal/assets/73314261/a3d04a1e-b277-4d60-b085-30ee2f9db82b)